docs/events/howto/remove_files_asynchronous.md

```diff
@@ -0,0 +1,114 @@
+Remove files asynchronous
```

Comment:
I guess that "asynchronous" should be "asynchronously" here and in following occurrences.

Reply:
Changed